### PR TITLE
Django-polls-app-updated_view_tempates_render

### PR DESCRIPTION
**Pull request by aadityaa-cuelogic**
Update template and views render method
Show 404 page if incorrect url is passed
Use template system to render DB data

### CODE REVIEW

File: firstprojectsite/polls/urls.py [hunk 1]
```diff
@@ -1,13 +1,22 @@
 from django.conf.urls import url
 from . import views
 
+# urlpatterns = [
```
Comment (ananthu-nair707) [inline firstprojectsite/polls/urls.py:4]: Remove commented code if not required,

File: firstprojectsite/polls/views.py [hunk 2]
```diff
@@ -1,22 +1,35 @@
...
+# way 1 by loader
+# def index(request):
+#     latest_question_list = Question.objects.order_by('-pub_date')[:5]
+#     template = loader.get_template('polls/index.html')
+#     context = {
+#         'latest_question_list': latest_question_list,
+#     }
+#     return HttpResponse(template.render(context, request))
+# way 2 by render
 def index(request):
```
Comment (ananthu-nair707) [inline firstprojectsite/polls/views.py:18]: Remove commented code if not required,

File: firstprojectsite/polls/views.py [hunk 3]
```diff
@@ -1,22 +1,35 @@
...
-    return HttpResponse(template.render(context, request))
+    context = {'latest_question_list': latest_question_list}
+    return render(request, 'polls/index.html', context)
+
+# def detail(request, question_id):
+#     try:
+#         question = Question.objects.get(pk=question_id)
+#     except Question.DoesNotExist:
+#         raise Http404("Question does not exist")
+#     return render(request, 'polls/detail.html', {'question': question})
```
Comment (ananthu-nair707) [inline firstprojectsite/polls/views.py:29]: Remove commented code if not required,

**Review by ananthu-nair707 — CHANGES_REQUESTED** [review]

File: firstprojectsite/polls/views.py (hunk 3 above)
Comment (aadityaa-cuelogic) [reply firstprojectsite/polls/views.py:29]: these are multiple ways to write same code..so kept commented code

File: firstprojectsite/polls/views.py (hunk 2 above)
Comment (aadityaa-cuelogic) [reply firstprojectsite/polls/views.py:18]: these are multiple ways to write same code..so kept commented code

File: firstprojectsite/polls/urls.py (hunk 1 above)
Comment (aadityaa-cuelogic) [reply firstprojectsite/polls/urls.py:4]: these are multiple ways to write same code..so kept commented code

**Comment by aadityaa-cuelogic** [comment]
these are multiple ways to write same code..so kept commented code

**Review by aadityaa-cuelogic — COMMENTED** [review]
these are multiple ways to write same code..so kept commented code

**Review by ananthu-nair707 — APPROVED** [review]

**Pull request merged**